dns/rfc2136/src/etc/inc/plugins.inc.d/rfc2136.inc

```diff
@@ -123,7 +123,7 @@ function rfc2136_configure_do($verbose = false, $int = '', $updatehost = '', $fo
         $keyfile = "/var/etc/nsupdatekey{$i}";
         $keyfill = [
             "key \"{$keyname}\" {",
-            "\talgorithm hmac-md5;",
+            "\talgorithm {$dnsupdate['keyalgo']};",
```

Comment:
IIRC keyalgo was used before so this does break setups as mentioned before.

Reply:
I do not know where keyalgo was used before or what it would break. Is this just a matter of changing the name of the variable for passing the selected key algorithm to the backend?

Reply:
Sorry, my mistake, the old one was "keytype" 989029fc909803 so this is fine

Reply:
There is an error here about missing key for when you did not save settings yet but let me fix it. I'm unsure about switching the default from md5 to sha512, if we add a selection and preselect the highest one wouldn't it have been better to not have the selection?